example/src/Bar.java

```diff
@@ -1,8 +1,6 @@
 package src;
 
 // Unused imports are suppressed in suppressions.xml, so this should not raise issue.
-import java.util.Objects;
```

Comment:
The comment implies these unused imports are intentional and shouldn't be removed?

Reply:
oh oops, I ran format to test this and should have checked for spurious deltas